spring-boot-data/src/main/java/com/baeldung/hexagonal/adapter/ProductController.java

```diff
@@ -0,0 +1,52 @@
+package com.baeldung.hexagonal.adapter;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.baeldung.hexagonal.domain.Product;
+import com.baeldung.hexagonal.port.ProductAPIPort;
+import com.baeldung.hexagonal.service.ProductService;
+
+/**
+ * 
+ * @author shwetaJ
+ *
+ */
+
+@RestController
```

Comment:
These changes don't relate to the article

Reply:
Yes, this file was for my evaluation article.